### PR TITLE
feat: adding super dash counter example

### PR DESCRIPTION
**Pull request by erickzanardo**
## Description

A simple sample using Flame.

It implements a classic counter, but gamified. Originally the character does not have enough power to jump and reach the controllers blocks, which we can fix with a patch.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 🧪 Tests
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: super_dash_counter/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
+
+For help getting started with Flutter development, view the
+[online documentation](https://docs.flutter.dev/), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```
Comment (felangel) [inline super_dash_counter/README.md]:
```suggestion
```

File: super_dash_counter/analysis_options.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
+
+linter:
+  rules:
```
Comment (felangel) [inline super_dash_counter/analysis_options.yaml]:
```suggestion
```

File: super_dash_counter/analysis_options.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
```
Comment (felangel) [inline super_dash_counter/analysis_options.yaml:1]: Also might want to use very_good_analysis to be consistent with the other shorebird projects 🤷

File: super_dash_counter/pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+version: 1.0.0+1
+
+environment:
+  sdk: '>=3.3.3 <4.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
+  cupertino_icons: ^1.0.6
```
Comment (felangel) [inline super_dash_counter/pubspec.yaml]:
```suggestion
```

Is that needed?

**Review by felangel — COMMENTED** [review]
Overall looks great but this might make more sense under your own personal GitHub since this repo has historically been for source code used in the shorebird docs guides 🤷 

wdyt @bryanoltman @eseidel?

**Comment by eseidel** [comment]
I don't have a strong opinion one way or the other about this specific example.  I would say if we have guidelines for what should/shouldn't be in samples we should make those clear in the README.md :) e.g.  We likely aren't accepting feature patches to samples in general, but would accept bug fixes.  If samples are unused by our team we will delete them.

File: super_dash_counter/analysis_options.yaml (hunk 3 above)
Comment (erickzanardo) [reply super_dash_counter/analysis_options.yaml:1]: Oh, I checked on the flavors one and I saw just the flutter lints, so I thought that was the standard for samples. I will def change

**Comment by erickzanardo** [comment]
Moved the code to https://github.com/erickzanardo/super_dash_counter

Closing this.

**Pull request closed without merging**